src/server/game/Entities/Creature/Creature.cpp

```diff
@@ -2529,12 +2604,19 @@ void Creature::UpdateMoveInLineOfSightState()
         m_moveInLineOfSightDisabled = false;
 }
 
-void Creature::SaveRespawnTime()
+void Creature::SaveRespawnTime(uint32 forceDelay)
```

Comment:
Cant be chrono?

Reply:
Probably, I will look later.

Reply:
Not able to fix without big refactor. I will look for another PR.